collections/ansible_collections/cloudkit/config_as_code/roles/aap/vars/eda.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Create the controller credential so EDA can trigger job templates
+eda_credentials:
+  - name: "{{ aap_eda_service_user_name }}"
+    organization: "{{ aap_organization_name }}"
+    credential_type: 'Red Hat Ansible Automation Platform'
+    inputs:
+      username: "{{ aap_eda_service_user_name }}"
+      password: "{{ aap_eda_service_user_password }}"
+      host: "{{ aap_hostname }}/api/controller/"
+
+# Create EDA project from the Git repo
+eda_projects:
+  - name: "{{ aap_prefix }}-eda"
+    organization: "{{ aap_organization_name }}"
+    url: "{{ aap_project_git_uri }}"
+    wait: true
+    interval: 10
+    sync: true
+
+# Create decision environment
+eda_decision_environments:
+  - name: "{{ aap_prefix }}-de"
+    organization: "{{ aap_organization_name }}"
+    image_url: "registry.redhat.io/ansible-automation-platform-25/de-supported-rhel9:latest"
```

Comment:
Do dependencies get installed at runtime? My assumption was that you'd build a custom decision environment so it would have the various dependencies already baked in.

Reply:
Ok now I see below that it looks like we're using ansible tooling to define and create a custom decision environment.

Reply:
there are 2 images:
- decision environment, used by EDA. Here we don't need to add any dependencies because it basically triggers jobs in the controller, so it's fine to use the default one (but I opened a ticket to get ours: https://github.com/innabox/cloudkit-aap/issues/13)
- execution environment, used by the controller, it's where the playbooks run in. I updated this one to embed all our dependencies.

Reply:
Ah that helps, thanks. I've been running `ansible-rulebook` directly instead of running AAP, and I guess it combines both of those concepts into one image.